data/remnant ships.txt

```diff
@@ -24,8 +24,9 @@ ship "Starling"
 		"drag" 3.8
 		"heat dissipation" 0.7
 		"fuel capacity" 900
+		"ramscoop" 1
 		"cargo space" 36
-		"outfit space" 312
+		"outfit space" 315
```

Comment:
This outfit space increase should be reverted across all ships due to the implementation of the scanners as built in outfits.

Reply:
reverted